src/api/UsersRepository.js

```diff
@@ -25,6 +25,10 @@ export default {
         return kcRequest().then(axiosInstance => axiosInstance.put(`${resource}/${userId}`, content));
     },
 
+    deleteUser(userId) {
+        return kcRequest().then(axiosInstance => axiosInstance.put(`${resource}/${userId}`))
+    },
+
```

Comment:
Delete this maybe? It's not the right verb, it it? `put`?

Reply:
it's been removed